server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java

```diff
@@ -68,7 +70,12 @@ public NettyTransport(InetSocketAddress address, ProtocolServerConfiguration con
 
       // Need to initialize these in constructor since they require configuration
       masterGroup = buildEventLoop(1, new DefaultThreadFactory(threadNamePrefix + "-ServerMaster"));
-      ioGroup = buildEventLoop(configuration.ioThreads(), new DefaultThreadFactory(threadNamePrefix + "-ServerIO"));
+      if (cacheManager != null) {
```

Comment:
is it possible for the cache manager to be `null`?

Reply:
Yes, unfortuantely. This is needed for the jcache tests as it uses the single port server thingy. I tried removing parts of it, but the new server uses too much of it.

Reply:
I have added a comment to it.